test/verify/check-services

```diff
@@ -188,6 +189,270 @@ WantedBy=default.target
+        b.wait_present("[data-content='week-days'][data-index=1]")
+        b.click("[data-index=1][data-content='week-days'] button")
+        b.click("[data-index=1][data-content='week-days'] .dropdown-menu li[value=7] a")
+        b.set_val("[data-index=1][data-content='hours']", "20")
+        b.set_val("[data-index=1][data-content='minutes']", "12")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('weekly_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-03-07 00:00:00'")
+        m.execute("systemctl start weekly_timer.timer")
+        b.wait_present(svc_sel('weekly_timer.timer'))
+        b.wait_in_text(svc_sel('weekly_timer.timer'), "Weekly timer")
+        self.assertIn("Sun 2020-03-08 20:12", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-03-09 00:00:00'")
+        b.wait_present(svc_sel('weekly_timer.timer'))
+        self.assertIn("Fri 2020-03-13 12:45", m.execute("LC_ALL=C systemctl list-timers"))
+        # checks if timer runs on next week's Friday and Sunday
+        m.execute("timedatectl set-time '2020-03-14 00:00:00'")
+        b.wait_present(svc_sel('weekly_timer.timer'))
+        self.assertIn("Sun 2020-03-15 20:12", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-03-17 00:00:00'")
+        b.wait_present(svc_sel('weekly_timer.timer'))
+        self.assertIn("Fri 2020-03-20 12:45", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop weekly_timer.timer")
+        # creates a new daily timer that runs at 2:40 and at 21:15 every day
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "daily_timer")
+        b.set_val("#description", "Daily timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=1440] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.set_val("[data-index=0][data-content='hours']", "2")
+        b.set_val("[data-index=0][data-content='minutes']", "40")
+        b.set_val("[data-index=1][data-content='hours']", "21")
+        b.set_val("[data-index=1][data-content='minutes']", "15")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('daily_timer.timer'))
+
+        m.execute("timedatectl set-time '2020-03-17 00:00:00'")
+        m.execute("systemctl start daily_timer.timer")
+        b.wait_present(svc_sel('daily_timer.timer'))
+        b.wait_in_text(svc_sel('daily_timer.timer'), "Daily timer")
+        self.assertIn("Tue 2020-03-17 02:40", m.execute("LC_ALL=C systemctl list-timers"))
```

Comment:
see above, probably a race condition (maybe it would be good to write a function to cover waiting for this info)

or is this one alleviated by starting it on the command line?